src/server/lib/tracking.ts

```diff
@@ -11,8 +18,8 @@ export const buildBannerCampaignCode = (test: BannerTest, variant: BannerVariant
 export const buildAmpEpicCampaignCode = (testName: string, variantName: string): string =>
     `AMP__${testName}__${variantName}`;
 
-export const buildGutterCampaignCode = (testName: string, variantName: string): string =>
-    `gutter__${testName}__${variantName}`;
+export const buildGutterCampaignCode = (testName: GutterTest, variantName: GutterVariant): string =>
+    `gutter__${testName.name}__${variantName.name}`;
```

Comment:
Wondering about the implication of changing the prefix to the campaign code to gutter-liveblog rather than just gutter to avoid changes if we add an article version later.  Any advice?